### PR TITLE
Use g0v.json schema to render fields

### PR DESCRIPTION
**Pull request by shelling**
g0v.json schema definition has been showed at github.com/g0v/g0v.json. This PR aims at using its version 1 definition to render to editor fields

### CODE REVIEW

**Comment by shelling** [comment]
oh, I forgot to modify the submit handler. let me refine it tomorrow. DO NOT merge it today.

**Comment by shelling** [comment]
ready to merge

File: index.js [hunk 1]
```diff
@@ -72,7 +73,7 @@ var app = new Vue({
             var branch
             var fork
             new Promise(function(out_resolve, reject) {
-                app.conn.post(`/repos/${app.repo}/forks`)
+                app.conn.post(`/repos/${app.schema.id.prefill}/forks`)
```
Comment (poga) [inline index.js]: so we can't prefill target repo with repo params now? maybe README should be updated too?

File: index.js [hunk 2]
```diff
@@ -96,14 +97,19 @@ var app = new Vue({
...
-                        type: "blob",
-                        sha: newBlob.sha
-                    }
-                ]})})
+                return app.conn.post(`/repos/${x.login}/${fork.name}/git/trees`, {
+                  data: JSON.stringify({
+                          base_tree: currentHEADtree.sha,
+                          tree: [
+                            {
+                              path: "g0v.json",
```
Comment (poga) [inline index.js]: use `filename` instead of hard-coded string?

File: index.js (hunk 1 above)
Comment (shelling) [inline index.js]: Maybe we can replace `id` with `repo` in schema definition. I defined `id` before seeing `repo` and they are actually the same thing.

File: index.js (hunk 2 above)
Comment (shelling) [inline index.js]: `filename` is `undefined` in my environment, quite weird. I actually prefer using `filename`. Maybe I can figure it out. poke you later

File: index.js (hunk 1 above)
Comment (shelling) [inline index.js]: Changed `id` to `repo` in g0v.json schema. You can prefill with `?repo=blah` now

File: index.js (hunk 2 above)
Comment (shelling) [inline index.js]: Changed back to `filename`

**Comment by shelling** [comment]
@poga Adjusted 2 places you mentioned. Ready to merge.

**Pull request merged**